README.md

```diff
@@ -1,2 +1,94 @@
-# triton-touchbase
-Touchbase stack designed for container-native deployment on Joyent's Triton platform
+triton-touchbase
+==========
+
+*[Touchbase](https://github.com/couchbaselabs/touchbase) stack designed for container-native deployment on Joyent's Triton platform*
+
+This repo is a demonstation of a multi-tier container-native application architecture on [Joyent's](https://www.joyent.com/) Triton platform. It uses Touchbase and its supporting services as an example. It includes the following components:
+
+- Consul, acting as a discovery service
```

Comment:
Obviously, this would have to be done in a followup commit, after we know the proper URLs.